app/ui/src/main/java/com/fsck/k9/ui/helper/MaterialColors.java

```diff
@@ -0,0 +1,49 @@
+package com.fsck.k9.ui.helper;
+
+public class MaterialColors {
```

Comment:
Can you cut down this class further? Unless I am mistaken, there's a bunch of data that is not being used.

Reply:
That shouldn't be a problem. Proguard probably removes the unused stuff anyways. I could remove the `SHADE_XY` arrays but I would prefer to fully keep the color arrays. That makes it much easier to use the class. Without the generic list of all colors, `ContactLetterBitmapCreator.kt ` would need to specify colors as list of integers like it was before. That is a lot harder to read and modify.

Reply:
Just had a look at the compiled apk. Each of the arrays has a size of around 10 bytes, so that's not a big problem.

Reply:
It's not about the APK size. It's about the code size you need to parse as a human.